docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -183,7 +183,6 @@ image::sak-create-countries-topic.png[Image of wizard to create a topic]
 * *Replicas*: For this release of {product-kafka}, the replicas are preconfigured. The number of partition replicas for the topic is set to `3` and the minimum number of follower replicas that must be in sync with a partition leader is set to `2`. Replicas are copies of partitions in a topic. Partition replicas are distributed over multiple brokers in the cluster to ensure topic availability if a broker fails. When a follower replica is in sync with a partition leader, the follower replica can become the new partition leader if needed.
 +
 After you complete the topic setup, the new Kafka topic is listed in the topics table for your Kafka instance. You can now run the Node.js application to start producing and consuming messages.
-
```

Comment:
@fmcdonal It looks like removing a line in this file was accidental?

Reply:
@jbyrne-redhat resolved

Reply:
@kstanley could you review at your convenience please? thanks